docs/en/Installation.md

```diff
@@ -37,4 +38,18 @@ SilverLeague\IDEAnnotator\DataObjectAnnotator:
     enabled_modules:
       - mysite
       - otherfolderinsiteroot
-````
+```
+
+If the module you want annotated, has it's own composer.json file, and a name declared, you can enable it like this:
+
+```yaml
+
+---
+Only:
+    environment: 'dev'
+---
+SilverLeague\IDEAnnotator\DataObjectAnnotator:
+    enabled_modules:
+      - mysite
+      - SilverLeague/IDEAnnotator

```

Comment:
is this the module name or the namespace?

Reply:
Module name as specified in the composer.json. There is no way of trusting the namespace.